frontend/src/pages/ChainMemberList.tsx

```diff
@@ -403,13 +419,25 @@ function ParticipantsTable(props: {
   }
 
   function onRemove() {
-    let chainUID = props.chain.uid;
```

Comment:
Place all required values as variables here, to ensure that nothing could change between *opening the dialogue* and *clicking executing the action*